doma-spring-boot-samples/doma-spring-boot-sample-docker-compose/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.application.name=doma-spring-boot-sample-docker-compose
+spring.sql.init.mode=always
+
+# Doma configuration
+doma.dialect=POSTGRES
+logging.level.org.springframework.jdbc.datasource.DataSourceTransactionManager=DEBUG
```

Comment:
`DataSourceTransactionManager` is old. Use `JdbcTransactionManager` instead.

`logging.level.org.springframework.jdbc.support.JdbcTransactionManager=DEBUG`